### PR TITLE
convert temperatures

### PR DESCRIPTION
**Pull request by innovia**

### CODE REVIEW

**Comment by boarik** [comment]
Forgot to mention this earlier, please follow this convention for PR names:
`[branch name] branch description`
e.g.
`[exercises\exercise_3] Convert temperatures`

File: 03-functions-and-loops/temperatures.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def convert_to_fahrenheit(celsius):
+    fahrenheit = float(celsius) * 9/5 + 32
```
Comment (boarik) [inline 03-functions-and-loops/temperatures.py]: Inline this in the same line with the return.

Comment (boarik) [inline 03-functions-and-loops/temperatures.py]: please add spaces like so: `9 / 5`

File: 03-functions-and-loops/temperatures.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    return fahrenheit
+
+def convert_to_celsius(fahrenheit):
+    celsisus = (float(fahrenheit) - 32) * 5/9
```
Comment (boarik) [inline 03-functions-and-loops/temperatures.py]: Same as above.

**Comment by boarik** [comment]
Round 1 done.

**Comment by innovia** [comment]
round 2 done

**Pull request merged**